module/icp/algs/skein/skein_block.c

```diff
@@ -57,11 +57,11 @@
 
 /* Skein_256 */
 #if	!(SKEIN_USE_ASM & 256)
-
 void
 Skein_256_Process_Block(Skein_256_Ctxt_t *ctx, const uint8_t *blkPtr,
     size_t blkCnt, size_t byteCntAdd)
-{				/* do it in C */
+{
+/* do it in C */
```

Comment:
Let's just drop the comment here which isn't useful.  These same issues interestingly appear in the OpenZFS version so they must somehow be excluding this file from their checks.  Still, let's fix it.